v-next/hardhat-build-system/src/internal/types/artifacts.ts

```diff
@@ -41,11 +36,6 @@ export interface Artifacts {
    */
   getBuildInfo(fullyQualifiedName: string): Promise<BuildInfo | undefined>;
 
-  /**
-   * Synchronous version of getBuildInfo.
-   */
-  getBuildInfoSync(fullyQualifiedName: string): BuildInfo | undefined;
```

Comment:
This is because there are no usages from the build system?

Reply:
Or is it because utils implies async to you are removing non-async versions?

Reply:
Yes, correct